libraries/core-react/src/components/TextField/InputWrapper.tsx

```diff
@@ -1,8 +1,101 @@
 import * as React from 'react'
-import { InputHTMLAttributes } from 'react'
+import { InputHTMLAttributes, ReactNode } from 'react'
 import { useTextField } from './context'
 import { Input } from '../Input'
+import { Icon } from './Icon'
 import type { Variants } from './types'
+import type { ComponentToken } from '@equinor/eds-tokens'
+import styled, { css } from 'styled-components'
+import { typographyTemplate, outlineTemplate } from '@utils'
+import * as tokens from './TextField.tokens'
+
+const { textfield } = tokens
+
+const Variation = ({
+  variant,
+  isFocused,
+  token,
+}: {
+  variant: string
+  token: ComponentToken
+  isFocused: boolean
+}) => {
+  if (!variant) {
+    return ``
+  }
+
+  return css`
+    box-shadow: ${() =>
+      isFocused
+        ? `none`
+        : variant === 'default'
+        ? `inset 0 -1px 0 0 ${
+            token.border?.type === 'border' && token.border?.color
+          }`
+        : `0 0 0 1px ${
+            token.border?.type === 'border' && token.border?.color
+          }`};
```

Comment:
Struggled a lot to find the easiest syntax for this 🔥  styled-components. Couldn't make it work without the func Might be a user error, but I gave up. 😓

Reply:
I just tested, and I couldn't not see any difference without the removed func, so I suggest we remove it to avoid unintended consequences in the future